test/jasmine/tests/hover_label_test.js

```diff
@@ -1299,3 +1299,27 @@ describe('Test hover label custom styling:', function() {
         .then(done);
     });
 });
+
+describe('ohlc hover interactions', function() {
+    var data = [{
+        type: 'candlestick',
+        open: [2, 2],
+        high: [3, 3],
+        low: [0, 0],
+        close: [3, 3],
+        x: [1, 2, 3],
+        y: [0.12345, 0.23456, 0.34567]
```

Comment:
Oops. Does y do anything for ohlc? This looks like somehow leftover from another test, though it does trigger and respond to the fixed bug correctly.